src/index.js

```diff
@@ -1,11 +1,11 @@
-const { Telegraf } = require('telegraf');
-require('dotenv').config();
+const { Telegraf } = require("telegraf");
+require("dotenv").config();
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
 
-bot.start((ctx) => ctx.reply('Welcome to your AI Assistant!'));
+bot.start((ctx) => ctx.reply("Welcome to your AI Assistant!"));
 
 bot.launch();
 
-process.once('SIGINT', () => bot.stop('SIGINT'));
-process.once('SIGTERM', () => bot.stop('SIGTERM'));
+process.once("SIGINT", () => bot.stop("SIGINT"));
```

Comment:
**praise**: Proper handling of process signals to ensure graceful shutdown of the bot.